setup/MySQL/convert_mysql.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+#Set database name
+if [ $# -eq 2 ]; then
+   MYDATABASE="atpdatabase"
+else
+   MYDATABASE=$3
+fi
+
+#Get MySQL username and password from the arguments
+USERNAME=$1
+PASSWORD=$2
+
+#Get PWD
+PWD=`pwd`
+
+echo "Creating $MYDATABASE"
+
+mysql -u $USERNAME -p$PASSWORD << EOF
```

Comment:
I'm receiving many `mysql: [Warning] Using a password on the command line interface can be insecure.` warnings. Is there a better way to do this?